### PR TITLE
Change nifcloud_lb specification

### PR DESCRIPTION
**Pull request by heriet**
##### SUMMARY

Change nifcloud_lb module specification like [ec2_elb](https://docs.ansible.com/ansible/latest/modules/ec2_elb_module.html) to [ec2_elb_lb](https://docs.ansible.com/ansible/latest/modules/ec2_elb_lb_module.html)

This PR is implemented following
- create load balancer
- set filter ip addresses
- register/deregister instance
- some refactor

not implemented
- update load balancer 
- delete load balancer
- configure health check/ ssl plicies / ssl certificates

This PR is destructive changes on nifcloud_lb, but I think that does not require drastic changes by users.

##### RELATED ISSUE

#25 

##### HOW TO VERYFY IT

see tests

### CODE REVIEW

File: library/nifcloud_lb.py [hunk 1]
```diff
@@ -50,28 +51,68 @@
...
         description:
-            - Target Load Balancer port number (required for registration)
+            - IP version (v4 or v6)
         required: false
+        default: 'v4'
+    accounting_type:
+        description:
+            - Accounting type (1: monthly, 2: pay per use)
+        required: false
+        default: 'v4'
```
Comment (fuku2014) [inline library/nifcloud_lb.py]: I think default of accounting_type is `'1'`.

File: library/nifcloud_lb.py [hunk 2]
```diff
@@ -50,28 +51,68 @@
...
+    policy_type:
+        description:
+            - Encryption policy type (standard or ats)
+        required: false
+        default: 'standard'
+    instance_ids:
+        description:
+            - List of Instance ID
+        required: true
```
Comment (fuku2014) [inline library/nifcloud_lb.py]: I think required of instance_ids is `false`.

File: library/nifcloud_lb.py (hunk 1 above)
Comment (heriet) [reply library/nifcloud_lb.py]: Thanks! simple mistake... fixed it.

File: library/nifcloud_lb.py (hunk 2 above)
Comment (heriet) [reply library/nifcloud_lb.py]: Similarly, simple mistake... fixed it.

**Pull request merged**